include/ReadoutCard/DmaChannelInterface.h

```diff
@@ -96,6 +96,9 @@ class DmaChannelInterface
   // Returns the number of dropped packets, as reported by the BAR
   virtual int32_t getDroppedPackets() = 0;
 
+  // Returns a counter for the times a link's Superpage FIFO was empty
+  virtual uint32_t getFifoEmptyCounter(int link) = 0;
```

Comment:
@sy-c is this ok for you? (This PR addresses https://alice.its.cern.ch/jira/browse/ORC-385)